src/AI/JakeMan.java

```diff
@@ -730,7 +731,7 @@ private boolean isDudeFree(GameMap gameMap, Unit unit, XYCoord xyc, boolean amAt
           if( unitMapFriendly.get(counter).containsKey(coord) )
             counterPower = unitMapFriendly.get(counter).get(coord);
           if( counterIsMeAndIAmPeaceful )
-            counterPower -= PEACEFUL_SELF_THREAT_RATIO * unit.getHPFactor();
+            counterPower -= (PEACEFUL_SELF_THREAT_RATIO * unit.getHPFactor()) / 10;
```

Comment:
Isn't HP Factor already [0, 1]? Why divide by 10 again?

Reply:
It got changed too, because I rage against IEE754